### DISCUSSION TITLE
How to Install osgearth Triton With vcpkg?

### DISCUSSION THREAD

**Discussion by pusia1** [pelicanmapping/osgearth#2339] [Support Q&A] [answered]
How to Install osgearth Triton With vcpkg?

I want to install osgearth and triton with vcpkg.

How do I configure the portfile of vcpkg?

**Comment by gwaldron** [comment, marked as answer]
I haven't tried it myself, but you can probably just find the `vcpkg_cmake_configure` command and add the defines you need. First you would need to change `-DOSGEARTH_BUILD_TRITON_NODEKIT=OFF` to `ON`. Then you probably need to set the location of the Triton includes and libraries using the `-D` option, including `TRITON_INCLUDE_DIR` and `TRITON_LIBRARY`. Let me know if that works.
